ui/shared/utils/constants.js

```diff
@@ -550,6 +550,42 @@ const ORDERED_VEP_CONSEQUENCES = [
     value: 'DUP',
     group: VEP_GROUP_SV,
   },
+  {
+    description: 'A translocation variant',
+    text: 'Translocation',
+    value: 'BND',
+    group: VEP_GROUP_SV,
+  },
+  {
+    description: 'A copy number polymorphism variant',
+    text: 'Copy Number Polymorphism',
+    value: 'CNV',
+    group: VEP_GROUP_SV,
+  },
+  {
+    description: 'A Complex SV',
+    text: 'Complex SV',
+    value: 'CPX',
+    group: VEP_GROUP_SV,
+  },
+  {
+    description: 'A reciprocal chromosomal translocation',
+    text: 'Reciprocal chromosomal translocation',
```

Comment:
this is a very long name to show - is this actually the best human readable name for this type of variant or is there something shorter we can use?

Reply:
I found a [literature](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/reciprocal-chromosome-translocation) uses `RT` as the abbreviation for `Reciprocal Translocation`.  Not sure if it can be used for `Reciprocal chromosomal translocation`.

Reply:
I'll ask Anne

Reply:
Confirmed `Reciprocal Translocation` is the same, so lets use that for the text and leave the long form in the description